### PR TITLE
Add taxClass attribute to cart and order

### PR DESCRIPTION
**Pull request by BirgitBader**
@ubep please check and merge

### CODE REVIEW

File: apps/data-types.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | billingAddress | [address](page:apps-data-types#address) | The billing address for a cart. |
 | shippingAddress | [address](page:apps-data-types#address) | The shipping address for a cart. |
 | lineItemContainer | [lineItemContainer](page:apps-data-types#lineitemcontainer) | Contains the line items of a cart. |
+| shippingData | object of [shippingData](page:apps-data-types#shippingdata) | The shipping data of the order, i.e. short info on shipping method and price.|
```
Comment (ubep) [inline apps/data-types.md]: The payment/shipping data of a cart or an order

File: apps/data-types.md [hunk 2]
```diff
@@ -237,12 +239,17 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | billingAddress | [address](page:apps-data-types#address) | The billing address for the order.  |
 | shippingAddress | [address](page:apps-data-types#address) | The shipping address for the order.  |
 | invoicedOn | string | The date/time the order was invoiced. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z`   |
+| partialyPaidOn | string | The date/time a part of the order was paid. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z`   |
 | deliveredOn | string | The date/time the order was delivered. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z`   |
+| partialyInvoicedOn | string | The date/time a part of the order was invoiced. Expressed according to ISO 8601. Example: `2015-11-04T08:42:49.000Z`   |
```
Comment (ubep) [inline apps/data-types.md]: I thought we will first add these attributes to the docu when the typo is fixed?

Comment (BirgitBader) [inline apps/data-types.md]: Well, as already mentioned yesterday, I was not up-to-date about some other changes on the API, so I decided to include it and change the type, once it's merged. I don't want to risk another "out-of-date"-ness :-)

File: apps/data-types.md (hunk 1 above)
Comment (BirgitBader) [inline apps/data-types.md]: Why? We have shippingData and paymentData as separate attributes.

**Pull request merged**